src/admin/config.yml

```diff
@@ -108,6 +108,12 @@ collections:
     label_singular: Commercial Operations page
     folder: src/commercial-ops
 
+  - <<: *collection_defaults
+    name: contributing
+    label: Contributing
+    label_singular: Contributing
```

Comment:
I think this should be a "Contributing page"

Reply:
Will raise a PR